File: RentIT-Frontend/src/model/review.ts
import {User} from "src/model/user";

export interface Review {
  // this is gathered when receiving data, not adding
  // this can be userEmail or productId
  targetId?: string;
  rating: number;
  message: string;
}

export enum TARGET {
  USER = 'user',
  PRODUCT = 'product',
}

File: RentIT/src/main/java/com/rentit/model/Review.java
package com.rentit.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Review {
    private String targetId;
    private int rating;
    private String message;
}
